### DISCUSSION TITLE
Keploy Extension Project

### DISCUSSION THREAD

**Discussion by Ajeett01** [keploy/keploy#1722] [Ideas]
In this discussion, I will present my work on Keploy Extension Development and welcome input on potential enhancements. I've been an active contributor to Keploy, having resolved a significant number of issues. Leveraging my experience in building VSCode Extensions, I've been actively working on creating extension versions of Keploy for various IDEs.

**Comment by Ajeett01** [comment]
[Screencast from 26-03-24 10:30:51 AM IST.webm](https://github.com/keploy/keploy/assets/122796321/cf8e1710-4a0b-4217-ab53-0d277bbb8e6f)
I have created the UI for different components , I also successfully integrated Keploy Test functionality , and working on intergrating others .
I will share a video with all that functionality working. 
@Sonichigo @Sarthak160

**Reply by Sonichigo** [reply]
Hey Ajeet! UI looks pretty good!

**Comment by Ajeett01** [comment]
> Hey Ajeet! UI looks pretty good!
Yes! Thanks.
